File: backend/src/main/java/com/shieldx/securities/dto/BookingResponse.java
package com.shieldx.securities.dto;

import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class BookingResponse {
    private Integer bookingId;
    private Integer userId;
    private Integer vipId;
    private Integer securityTypeId;
    private String bouncerCount;
    private LocalDate startDate;
    private LocalDate endDate;
    private LocalTime startTime;
    private LocalTime endTime;
    private String location;
    private String status;
    private double totalPrice;
    private List<Integer> bouncerIds;
}

File: frontend/shieldx-frontend/src/app/dto/booking-response.ts
export interface BookingResponse {
  bookingId: number;
  userId: number;
  vipId: number;
  securityTypeId: number;
  bouncerCount: string;
  startDate: string; // Represents Java LocalDate (e.g., "2025-07-13")
  endDate: string;   // Represents Java LocalDate
  startTime: string; // Represents Java LocalTime (e.g., "14:30:00")
  endTime: string;   // Represents Java LocalTime
  location: string;
  status: string;
  totalPrice: number;
  bouncerIds: number[];
}
